### ISSUE TITLE
Have a list of uploaded apps available

### ISSUE THREAD

**Issue by LukasReschke** [nextcloud/appstore#16]
The home page should show the categories as well as the list of applications that are available. If somebody clicks on an application more details about it should be shown.

Probably best to solve this completely with some JS magic and not use any Python logic for this.

**Comment by adsworth** [comment]
isn't the title of the issue misleading? What do the installed apps have todo with the available apps and categories?

**Comment by BernhardPosselt** [comment]
@LukasReschke we need some JS guys for this. If we go for Python and HTML, it's 3 lines of Python code, one line for a route and an html template :)

**Comment by BernhardPosselt** [comment]
Something like this:

```python
from django.views.generic.list import ListView
from nextcloudappstore.core.models import App

AppsView(ListView):
    model = Apps

# urls for the homepage
url(r'^apps//?$', AppsView.as_view(), name='apps)
```

**Comment by LukasReschke** [comment]
> @LukasReschke we need some JS guys for this. If we go for Python and HTML, it's 3 lines of Python code, one line for a route and an html template :)

If it is that easy in Django I'm happy to also go the HTML and Django route :)

**Comment by adsworth** [comment]
Closed via #20
